adding about us hero section

tsvetelinak0 opened the pull request:
### What change does this PR introduce?
about us hero as component
added styles for hero-component
added styles for navbar - bg color only
background-color of body removed
fixed header section in about us page
added CopyPlugin to webpack - copy files and dirs that already exist to the build directory

### Notes

Run `npm run build`

### Ticket

- [Trello Ticket](https://trello.com/c/lEx5Ekwe/75-about-us-hero-section)

### Screenshots
**Mobile**
<img width="482" alt="Screenshot 2020-06-03 at 20 53 05" src="https://user-images.githubusercontent.com/27692549/83683430-4dcf2d00-a5dd-11ea-9cfc-e3c329b8c8cb.png">

**Tablet**
<img width="706" alt="Screenshot 2020-06-03 at 20 52 47" src="https://user-images.githubusercontent.com/27692549/83683451-57f12b80-a5dd-11ea-9bae-fb36344f07e8.png">

**Desktop**
<img width="1211" alt="Screenshot 2020-06-03 at 20 52 31" src="https://user-images.githubusercontent.com/27692549/83683477-60e1fd00-a5dd-11ea-913d-7ff979c11133.png">

lili2311 commented:
@tsvetelinak0 thank you for adding a screenshot! Please could you also share what it look like on mobile so we can compare to the [wireframes](https://www.figma.com/file/LcuMvUJmLSrCEB0Qy3D4BDIM/Sharespots?node-id=0%3A1)?

lili2311 commented on src/styles/components/_title.scss:
```diff
@@ -5,11 +5,11 @@
 @import "base/variables";
 
 .homepage_section_block {
+	background-color: $color-pink;
```
weird spacing here? please align all properties, do you have a SASS plugin installed in vscode?

tsvetelinak0 replied:
will align! def having sass plugin.. spacing might come from copy-paste

lili2311 commented on src/styles/styles.scss:
```diff
@@ -37,6 +36,62 @@ html, body {
   height: 100%;
 }
 
+.index_bottom_container {
```
could you please make this a component? This can be called a hero-section component and add all the styles inside there. We want to follow a component based design with a mobile first approach.

tsvetelinak0 replied:
sure will do!

lili2311 commented on src/styles/styles.scss:
```diff
@@ -37,6 +36,62 @@ html, body {
+  // background-color: black;
+  width: 100%;
+  height:250px;
+  // padding: 20px;
```
please remove any commented out styles, if they are not needed we don't need to keep them

lili2311 reviewed (changes requested):
Please see comment, just a little tidy up to do

lili2311 commented:
Design tweaks:
- please remove the border from the buttons, our button in designs is all flat and 1 colour with no borders.
- please lowecrase `us` in `About us` button as per design.

tsvetelinak0 commented:
@lili2311 it is not very clear to me from the designs if we see that `menu img` in the about-hero-section on tablets and mobiles?

tsvetelinak0 committed:
aboutushero as component, solved merge conflicts, added CopyPlugin

solved merge conflicts

fix aboutushero

added copy plugin to webpack to copy static files into dist folder

minor changes

lili2311 commented:
The image should be gone on table & mobiles just like wireframes show

lili2311 commented on src/styles/components/_title.scss:
```diff
@@ -7,9 +7,14 @@
 .homepage_section_block {
 	align-items: center;
     height: 450px;
+<<<<<<< HEAD
```
looks like accidental merge conflict commitetd?

lili2311 commented on src/styles/components/_hero_section.scss:
```diff
@@ -0,0 +1,50 @@
+// Variables
+//
+
+@import "base/variables";
```
do we normally import them here? I think having it in the root styles.scss should be enough before all other imports.

lili2311 commented on src/styles/styles.scss:
```diff
@@ -37,6 +38,24 @@ html, body {
   height: 100%;
 }
 
+<<<<<<< HEAD
```
please have a look at unresolved merge conflict here

lili2311 commented on templates/core/index.html:
```diff
@@ -33,4 +33,19 @@ <h2>Recommended venues:</h2>
 		{% endfor %}
 	</ul>
 </section>
+<<<<<<< HEAD
```
another one here 🙏

abu271 commented on src/styles/components/_title.scss:
```diff
@@ -1,15 +1,8 @@
-
 .homepage_section_block {
 	align-items: center;
     height: 450px;
```
@tsvetelinak0 Small format issue, this line is missing an extra space.

abu271 reviewed (commented):
Hi @tsvetelinak0, I ran the branch locally and viewed the page in mobile view and the footer height seems a bit large. Can we make the height smaller?
![Screenshot from 2020-06-10 21-50-12](https://user-images.githubusercontent.com/38472867/84318302-1c6ada00-ab66-11ea-81bf-5a3188c8a410.png)

lili2311 commented on templates/core/index.html:
```diff
@@ -33,4 +33,16 @@ <h2>Recommended venues:</h2>
+		</button>
+	</div>
+	<div class="hero_image">
+		<img class="menu_image" src="https://trello-attachments.s3.amazonaws.com/5b5deb66b4e3542422dbb2fd/5cc619deb2236e1f7be917f5/0d599e4dd34fa26912b2a1084adf7904/menu_(1).svg" alt="menu">
```
Please have a read https://docs.djangoproject.com/en/3.0/howto/static-files/

- We can create a folder `src/img` and add images there (nicely named!)
- add some changes to webpack to copy the files from `src/img` to `dist/img` look at webpack Copy tasks
and that will copy our images from `src` straight to static folder

lili2311 reviewed (dismissed):
Please change how we serve the images for this as we can't rely on the trello image to always be there if we delete the ticket.

tsvetelinak0 committed:
Merge branch 'feature/aboutushero' into develop

louiechristie reviewed (approved):
I approve this, Tsveti demonstrated it working to me.
